src/main/java/seedu/address/model/item/Item.java

```diff
@@ -0,0 +1,98 @@
+package seedu.address.model.item;
+
+import java.util.Objects;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+/**
+ * Represents an Item in FoodREM.
+ * TODO: Implement Item with Tags
+ */
+public class Item {
+    public final ItemName name;
```

Comment:
Note: There may be a merge conflict with the way names are implemented in tags. 
Should we have ItemName and TagName?
Or a name class used by both items and tags?
Or an abstract name class where itemName and TagName extends from it?

Reply:
I feel that there can be an ItemName and TagName because the items may have different requirements, keeping them separately defined might keep them flexible

Reply:
NOTE: As discussed on 5/10/22 meeting, we will likely have a common Name class shared with Tags class. Changes to this should be KIV, made as an issue.

Reply:
Never mind, we will split it for now

Reply:
This is due to validation being done differently for Items and Tags